src/main/scala/com/exasol/spark/util/ExasolConnectionManager.scala

```diff
@@ -56,21 +56,41 @@ final case class ExasolConnectionManager(config: ExasolConfiguration) {
       config.password
     )
 
-  def initParallel(mainConn: EXAConnection): Int =
-    mainConn.EnterParallel(config.max_nodes)
+  /**
+   * Starts a parallel sub-connections from the main JDBC connection.
+   *
+   * @param mainConnection the main connection
+   * @return the number of parallel connections
+   */
+  def initParallel(mainConnection: EXAConnection): Int =
+    mainConnection.EnterParallel(config.max_nodes)
 
-  def subConnections(mainConn: EXAConnection): Seq[String] = {
-    val hosts = mainConn.GetSlaveHosts()
-    val ports = mainConn.GetSlavePorts()
+  /**
+   * Returns the list of all parallel sub-connection URLs.
+   *
+   * @param mainConnection the main connection
+   * @return the list of sub-connections URLs
+   */
+  def subConnections(mainConnection: EXAConnection): Seq[String] = {
+    val hosts = mainConnection.GetWorkerHosts()
+    val ports = mainConnection.GetWorkerPorts()
+    val token = mainConnection.GetWorkerToken()
     hosts
       .zip(ports)
       .zipWithIndex
       .map {
         case ((host, port), idx) =>
-          s"jdbc:exa-slave:$host:$port;slaveID=$idx;slavetoken=${mainConn.GetSlaveToken()}"
+          s"jdbc:exa-worker:$host:$port;workerID=$idx;workertoken=$token"
```

Comment:
Do we need to check for jdbc driver version here? I guess this jdbc connection string format is new in 7.0.* right?

Reply:
Hey @3cham,

Yes, that is correct. The api changed in the 7.0.0 version of the Exasol database. But, it is backward compatible, I have already confirmed this with interface developers.

I have also updated travis matrix to include 6.2.9 and 7.0.2 versions so that we check against last two release series.